README.md

```diff
@@ -9,45 +9,176 @@ Please create a branch called `homework`. Then, create a file named `homework.js
   <br>
     <code>summation(5) // should return 15 because 1+2+3+4+5=15</code>
   </li>
+//My answer:
+
+const sum = function(n){
+  let a =0 ;
+for(let i = 1;i <= n ; i++){
+	a=a+i
+}
+return a;
+}
+console.log(sum(5));
+---------------------------------------------------------
   <li>Create a program to get the sum of all the even numbers in a group
   <br>
     <code > summationEven(5) // should return 6 because 2+4=6</code>
   </li>
-    
+//My answer:
+const susummationEven = function(n){
+  let a = 0 ;
+
+  for(let i = 0 ; i <= n ; i++ ){
+  if(i%2 ==0)
+{
+  a = a + i;	
+} 
+  }
+   return a;
+}
+console.log(susummationEven(5))
+---------------------------------------------------------
   <li>Create a function to get the average of a group of numbers 
   <br>
     <code> avg([8, 2, 2, 4]) // should return 4</code>
   </li>
+//My answer:
+
+const average = function(avg){
+let sum=0;
+let av = 0;
+  for(let i = 0 ; i < avg.length; i++ ){
+     sum=sum + avg[i];
+  }
+  av=sum/(avg.length)
+ console.log(av);
+}
+const avg=[8, 2, 2, 4];
+average(avg)
+---------------------------------------------------------
   <li>Create a function to reverse the letters in a word
   <br>
     <code > reverse("caterpillar") // should return "rallipretac"</code>
   </li>
+//My answer:
+
+const rev =function(text){
+text = text.split("")
+text = text.reverse()
+text = text.join("")
+return text
+}
+rev("caterpillar");
+
+//"rallipretac"
+---------------------------------------------------------
   <li>Create a function that takes an array of words and combines them with a dash
   <br>
     <code > addDashes(['test1', 'test2', 'test3']) // should return "test1-test2-test3"</code>
   </li>
+//My answer:
+
+const addDashes =function(text){
+text = text.join("-")
+return text
+}
+addDashes(['test1', 'test2', 'test3']);
+
+
+//"test1-test2-test3"
+---------------------------------------------------------
   <li>Function that will count up to a number and back down and return a string of the climb
   <br>
     <code> countUpAndDown(3) // should return "1 2 3 2 1"</code>
   </li>
+//My answer:
+
+const countUpAndDown = function(number){
+    for (let i = 1; i < number; i++) {
+        console.log(i);        
+    } 
+    for (i = number; i > 0; i--) {
+        console.log(i);
+    }
+}
+countUpAndDown(3)
+/*
+1
+2
+3
+2
+1
+*/
+---------------------------------------------------------
   <li>Write a function that will tell you all of the words in an array that contain the letter `a`
   <br>
     <code > wordsWithA(['cat', 'rabbit', 'dog', 'frog']) // should return ['cat', 'rabbit']</code>
   </li>
+//My answer:
+
+const wordsWithA =function(text){
+let a = text.pop()
+a = text.pop()
+return text;
+}
+wordsWithA(['cat', 'rabbit', 'dog', 'frog']);
+
+// ["cat", "rabbit"]
```

Comment:
this wont solve the problem.
to solve the problem:
- create an empty array to hold the words that contain the letter a.
- loop through the words array and check  each words if it contain the letter a using `includes()`  function.
- if true use `push()` to push the word into the empty array.
- return the array